tools/urlEditor/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "urlbuilder",
   "version": "1.0.0",
   "description": "",
-  "main": "dist/index.js",
+  "main": "src/index.ts",
```

Comment:
When I cloned the repo and tried to do the build, I got errors for these files since they weren't built yet. When I changed to point to the .ts files, as suggested in the error, everything worked fine.